### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by erikeelde**

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #Mon Nov 30 17:02:27 SGT 2020
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https://services.gradle.org/distributions/gradle-6.8.2-bin.zip
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0-bin.zip
```
Comment (erikeelde) [inline gradle/wrapper/gradle-wrapper.properties:4]: This change had some implications for publishing for me - the gradle publishing plugins have changed. I think the bintray plugin will work with it. But might be nice to test it out first.

File: lib/build.gradle.kts [hunk 2]
```diff
@@ -29,13 +28,11 @@ plugins {
 }
 
 android {
-    compileSdkVersion(30)
+    compileSdk = 30
 
     defaultConfig {
-        minSdkVersion(21)
-        targetSdkVersion(30)
-        versionCode = 7
```
Comment (marcelpinto) [inline lib/build.gradle.kts:37]: What's the reason of removing the versionCode + Name?

File: lib/build.gradle.kts [hunk 3]
```diff
@@ -119,7 +114,7 @@ val androidHtmlJar by tasks.register<Jar>("androidHtmlJar") {
 
 val androidSourcesJar by tasks.register<Jar>("androidSourcesJar") {
     archiveClassifier.set("sources")
-    from(android.sourceSets.getByName("main").java.srcDirs)
+    from(android.sourceSets.getByName("main").java.srcDirs())
```
Comment (marcelpinto) [inline lib/build.gradle.kts:117]: note, that this uses Kotlin Gradle files. Using the srcDirs directly is the normal kotlin practice

**Review by marcelpinto — CHANGES_REQUESTED** [review]
Thanks for doing this. Just a minor comment.

File: lib/build.gradle.kts (hunk 3 above)
Comment (marcelpinto) [reply lib/build.gradle.kts:117]: Oh I see that in the new version of gradle this changed.

**Review by marcelpinto — APPROVED** [review]

**Pull request merged**